src/roboptim-core-plugin-knitro.cc

```diff
@@ -25,76 +24,14 @@
 
 #include "roboptim-core-plugin-knitro.hh"
 
+using namespace std;
+
```

Comment:
We try not to abuse this. FYI, this should never be done in a header, so here that's ok. I'm just not sure it's that relevant (since we're not supposed to `std::cout` much here, or even not at all). It's often better to use this in a method's scope.